queries.sql

```diff
@@ -7,4 +7,26 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN  10.5 AND 17.3;
+UPDATE animals SET species = 'unspecified';
```

Comment:
- [ ] They update function should be made inside the transaction that start with `BEGIN` that has a `COMMIT` and the `ROLLBACK` to come back 👍 , Kindly ensure this update fucttion follow these steps 👍 

```sql
BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```